README.md

```diff
@@ -18,13 +18,13 @@
 
 </div>
 
-***Shifu*** is a [Kubernetes](https://kubernetes.io) native framework designed to abstract out the complexity of interacting with IoT devices. ***Shifu*** aims to achieve TRUE plug-and-play IoT device virtualization, management, control, and automation.
+***Shifu*** is a native framework to help IoT developers interact with physical devices. ***Shifu*** virtualizes the management and monitoring of devices, giving them the ability to run automatically. It solves the troubles of protocol and driver integration and enables devices to be plug-and-play.
```

Comment:
`native` => `kubernetes native` 
Native in computer science usually means it is written in C/C++